cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -1397,185 +1437,176 @@ def test_concurrent_export_and_cleanup(self):
+                    side_effect(sleep, 1),
                 )
 
                 mock_rq_get_current_job.return_value = MagicMock(timeout=5)
 
-                exited_by_timeout = False
-                try:
-                    export(dst_format=format_name, task_id=task_id)
-                except _LockTimeoutError:
-                    # should come from waiting for clear_removed_the_file
-                    exited_by_timeout = True
-
-                assert exited_by_timeout
-                mock_os_replace.assert_called_once()
-
+                export(dst_format=format_name, task_id=task_id)
+                mock_os_replace.assert_not_called()
 
         def _clear(*_, file_path: str, file_ctime: str):
             from os import remove as original_remove
-            from cvat.apps.dataset_manager.util import LockNotAvailableError
+
+            from cvat.apps.dataset_manager.views import FileIsBeingUsedError
 
             with (
-                patch('cvat.apps.dataset_manager.views.EXPORT_CACHE_LOCK_TIMEOUT', new=5),
+                patch("cvat.apps.dataset_manager.views.EXPORT_LOCK_TTL", new=5),
+                patch("cvat.apps.dataset_manager.views.EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT", new=10),
+                patch(
+                    "cvat.apps.dataset_manager.views.get_export_cache_lock",
+                    new=self.patched_get_export_cache_lock,
+                ),
+                patch(
+                    "cvat.apps.dataset_manager.views.os.remove", side_effect=original_remove
+                ) as mock_os_remove,
                 patch(
-                    'cvat.apps.dataset_manager.views.get_export_cache_lock',
-                    new=patched_get_export_cache_lock
+                    "cvat.apps.dataset_manager.views.rq.get_current_job"
+                ) as mock_rq_get_current_job,
+                patch("cvat.apps.dataset_manager.views.django_rq.get_scheduler"),
+                patch(
+                    "cvat.apps.dataset_manager.views.TTL_CONSTS",
+                    new={"task": export_outdated_after},
                 ),
-                patch('cvat.apps.dataset_manager.views.os.remove') as mock_os_remove,
-                patch('cvat.apps.dataset_manager.views.rq.get_current_job') as mock_rq_get_current_job,
-                patch('cvat.apps.dataset_manager.views.django_rq.get_scheduler'),
-                patch('cvat.apps.dataset_manager.views.TTL_CONSTS', new={'task': timedelta(seconds=0)}),
             ):
-                mock_os_remove.side_effect = chain_side_effects(
-                    side_effect(wait_condition, export_created_the_file),
-                    original_remove,
-                    side_effect(set_condition, clear_removed_the_file),
-                )
-
                 mock_rq_get_current_job.return_value = MagicMock(timeout=5)
 
-                exited_by_timeout = False
+                set_condition(clear_process_has_run)
```

Comment:
Probably, this line should be moved into a call inside of `clear_export_cache` for correctness.